src/payment/PaymentPage.jsx

```diff
@@ -140,6 +146,12 @@ class PaymentPage extends React.Component {
             'transaction-declined-message': (
               <TransactionDeclined />
             ),
+            'capture-key-2mins-message': (
+              <CaptureKeyTimeoutTwoMinutes />
+            ),
+            'capture-key-1min-message': (
+              <CaptureKeyTimeoutOneMinute />
+            ),
```

Comment:
Is `CaptureKeyTimeoutExplanation` not added here in the Message Codes because it's not being called in the sagas? It tags along with both 2mins and 1min message right?

Reply:
Correct; in fact `CKTExplanation` probably doesn't really need to be exported